app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/Database/LapCounterDatabase.java

```diff
@@ -1,24 +1,30 @@
-package edu.drexel.lapcounter.lapcounter.backend.Database.Workout;
-
-import android.arch.persistence.room.*;
+package edu.drexel.lapcounter.lapcounter.backend.Database;
 
+import android.arch.persistence.room.Database;
+import android.arch.persistence.room.Room;
+import android.arch.persistence.room.RoomDatabase;
 import android.content.Context;
 
+import edu.drexel.lapcounter.lapcounter.backend.Database.Device.Device;
+import edu.drexel.lapcounter.lapcounter.backend.Database.Device.DeviceDao;
+import edu.drexel.lapcounter.lapcounter.backend.Database.Workout.Workout;
+import edu.drexel.lapcounter.lapcounter.backend.Database.Workout.WorkoutDao;
 
-@Database(entities = {Workouts.class}, version = 1)
-public abstract class WorkoutDatabase extends RoomDatabase {
 
+@Database(entities = {Workout.class, Device.class}, version = 1)
+public abstract class LapCounterDatabase extends RoomDatabase {
 
-    public abstract WorkoutDao workoutDao();
 
+    public abstract WorkoutDao workoutDao();
+    public abstract DeviceDao deviceDao();
 
-    private static volatile WorkoutDatabase INSTANCE;
+    private static volatile LapCounterDatabase INSTANCE;
 
-    static WorkoutDatabase getDatabase(final Context context) {
+    public static LapCounterDatabase getDatabase(final Context context) {
         if (INSTANCE == null) {
-            synchronized (WorkoutDatabase.class) {
+            synchronized (LapCounterDatabase.class) {
                 if (INSTANCE == null) {
-                    INSTANCE = Room.databaseBuilder(context.getApplicationContext(), WorkoutDatabase.class, "workoutsdb").build();
+                    INSTANCE = Room.databaseBuilder(context.getApplicationContext(), LapCounterDatabase.class, "lapcounterdb").build();
```

Comment:
I'd recommend putting `"lapcounterdb"` in a string constant just in case anything else needs to reference the file someday.